osm2streets/src/road.rs

```diff
@@ -19,13 +19,27 @@ pub struct Road {
 
     pub src_i: IntersectionID,
     pub dst_i: IntersectionID,
+
+    /// The OSM `highway` tag indicating the type of this road. See
+    /// <https://wiki.openstreetmap.org/wiki/Key:highway>.
+    ///
+    /// Note for railways, this is actually the `railway` tag instead.
+    pub highway_type: String,
```

Comment:
We could do something more type-safe eventually like https://github.com/a-b-street/osm2lanes/blob/main/osm-tag-schemes/src/highway.rs, but I'd vote to save that for osm2lanes cutover